mod.yaml

```diff
@@ -3,12 +3,12 @@
 Metadata:
 	Title: Over Powered Mod
 	Description: Crazy modifications to Red Alert by Boolbada
-	Version: 007-Playtest-20170304
+	Version: {DEV_VERSION}
 	Author: Boolbada
 
 RequiresMods:
-	#modchooser: playtest-20170304
 	modchooser: {DEV_VERSION}
+	as: {DEV_VERSION}
```

Comment:
Don't do this! I understand your intention but this makes the whole mod required and not just the folder and the dll!